docs/guides/integrating-optimize.md

```diff
@@ -0,0 +1,118 @@
+---
+id: integrating-optimize
+title: Integrating Optimize
+sidebar_label: Integrating Optimize
+---
+
+## Purpose
+
+The following document provides a basic end-to-end glance into Optimize and its features for new business users.
+
+Optimize offers business intelligence tooling for Camunda enterprise customers. By leveraging data collected during process execution, you can access reports, share process intelligence, analyze bottlenecks, and examine areas in business processes for improvement.
+
+With Optimize, review heatmap displays for a closer look at the number of instances that took longer than average, based on duration distribution. As a result, reap actionable insights and rapidly identify the constraints of your system.
+
+See [What is Optimize](https://camunda.com/products/camunda-platform/optimize/reports/) for an in-depth overview of Optimize’s capabilities.
```

Comment:
Reports are only one feature on Optimize. I would prefer them going to the higher level page
https://camunda.com/products/camunda-platform/optimize/

Reply:
Agreed here -- adjusted.